README.md

```diff
@@ -207,6 +207,17 @@ Why is this style of upgrading good?  A couple reasons: Everyone is starting fro
 
 To reconfigure Pi-hole you'll either need to use an existing container environment variables or if there is no a variable for what you need, use the web UI or CLI commands.
 
+### Building an image with alternative component branches
+
+Occasionally you may need to try an alternative branch of one of the components (`core`,`web`,`ftl`). On bare metal you would run, for example, `pihole checkout core custombranchname`, however in Docker world we have disabled this command as it can cause unpredictable results.
```

Comment:
Where did we disable the checkout command?

Reply:
This was actually reverted here: https://github.com/pi-hole/docker-pi-hole/pull/1101

Reply:
Then Promo should decide if the want re-revert the change or change the Readme.

Reply:
#1101 was a v5 image change - I guess the `unsupportedFunc` stuff disappeared somewhere else for the v6 image as it was pretty much built from the ground up. I will add it in this PR

Reply:
Ah, it's because I got rid of `install.sh` in it's entirety.

Reply:
Maybe the function is better placed in the `core` repo. Just check for the presence of the docker tag before executing the functions?

Reply:
We could add a check at the beginning of each function in the `core` repo to exit early if the docker tag is detected. That would achieve the same thing I think, at least here it is minimal patching and makes contextual sense?

Reply:
We should not rely on my memory. I had entirely forgotten about this:

https://github.com/pi-hole/pi-hole/pull/5018

So to answer @yubiuser's original question of "Where did we disable the checkout command?" ... In the above PR. I'll force away the second commit here

Reply:
And also I should read better. Checkout still needs to be disabled the same way. IGNORE EVERYTHING I AM WRITING. I'M HUNGOVER.

Reply:
https://github.com/pi-hole/pi-hole/pull/5416